src/compas/artists/robotmodelartist.py

```diff
@@ -362,19 +373,21 @@ def update_tool(self, joint_state=None, visual=True, collision=True, transformat
 
         Parameters
         ----------
-        joint_state : :obj:`dict`or :class:`compas.robots.Configuration`, optional
+        joint_state : dictor :class:`compas.robots.Configuration`, optional
             A dictionary with joint names as keys and joint positions as values.
             Defaults to an empty dictionary.
         transformation : :class:`compas.geometry.Transformation`, optional
             The (absolute) transformation to apply to the entire tool's geometry.
-            If ``None`` is given, no additional transformation will be applied.
-            Defaults to ``None``.
+            If None is given, no additional transformation will be applied.
```

Comment:
why the removal of the keyword styling?

Reply:
i stopped doing :obj:`int` and :obj:`float` etc because it clutters the actual docstring and doesn't really add anything useful in the rendered version (in my opinion, but that is of course debatable).

similarly (and following the numpy style guide), i stopped adding those backticks on None, True, False... but also debatable